docs/src/assets/scripts.js

```diff
@@ -178,7 +179,9 @@ const getWebpage = async (folder, hash, shallUpdateHistory) => {
     searchInput.focus();
     const searchResultsList = doc.getElementById('search-results-list');
     searchInput.addEventListener('keyup', async (evt) => {
-      const { hits } = await searchIndex.search(searchInput.value);
+      const { hits: searchHits } = await searchIndex.search(searchInput.value);
+      const { hits: discussionHits } = await discSearchIndex.search(searchInput.value);
+      const hits = [ ...searchHits, ...discussionHits ];
```

Comment:
If they are combined, then you won't be putting all of the discussions at the end

Reply:
[504dd82](https://github.com/reach-sh/reach-lang/pull/1113/commits/504dd82ab4c7058b4a69ac651252dd97b3ef6151)

Reply:
https://github.com/reach-sh/reach-lang/pull/1113/commits/786581b9769b35184cccfaf16530fbcfb20b19dd#diff-492c64361dcbc9c0a47af787344fde877166ef1fecc66716d69710e201f666f9R63